packages/api/src/resolvers/article/index.ts

```diff
@@ -822,8 +822,12 @@ export const bulkActionResolver = authorized<
         },
       })
 
-      // the query size is limited to 255 characters
-      if (!query || query.length > 255) {
+      // the query size is limited to 1000 characters
```

Comment:
hah! I was wondering if it was something like that. I think we might need it even higher if we are allowing 100 uuids.